src/components/modules/weeklyBanner/WeeklyBanner.tsx

```diff
@@ -0,0 +1,319 @@
+import React, { useState } from 'react';
+import { css } from 'emotion';
+import { ThemeProvider } from 'emotion-theming';
+import { body, headline, textSans } from '@guardian/src-foundations/typography/cjs';
+import { neutral } from '@guardian/src-foundations/palette';
+import { from, until } from '@guardian/src-foundations/mq';
+import { space } from '@guardian/src-foundations';
+import { Button, LinkButton, buttonReaderRevenue } from '@guardian/src-button';
+import { brand } from '@guardian/src-foundations/themes';
+import Logo from '../guardianLogo/Logo';
+import Close from '../closeButton/Close';
```

Comment:
I was just looking at the `<Close />` component at `components/modules/closeButton/Close.tsx` and wondered whether we need this component, could we not just `import { SvgClose } from '@guardian/src-icons'` and use `<SvgClose />` in it's place?

Reply:
Yes, definitely. Thanks for pointing that out @GHaberis